docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -0,0 +1,198 @@
+  - meaning there's always one active and one passive region for serving active user traffic
+  - serving traffic to both regions will result in a detachment of the WebApps and users potentially observing different data in Operate and Tasklist
+- The customers operating their Camunda 8 setup are responsible for detecting a regional failure and executing the [operational procedure](<-- TODO: link -->)
+- Identity is not supported
+  - Multi-tenancy does not work
+  - Role Based Access Control (RBAC) does not work
+- Optimize is not supported
+  - This is due to Optimize depending on Identity to work
+- Connectors are not supported
+  - This is due to Connectors depending on Operate to work for inbound connectors and potentially resulting in race condition
+- During the fallback procedure, there’s a small chance that some data will be lost in Elasticsearch affecting Operate and Tasklist
+  - This **does not** affect the processing of process instances in any way. The impact is that some information about the affected instances might not be visible in Operate and Tasklist.
+  - This will be further explained in the [operational procedure](<!-- TODO: Link -->) during the relevant step.
+- Zeebe cluster scaling must be disabled
+  - Zeebe cluster size (broker count) must be static in size
+- Web-Modeler is a standalone component and is not covered in this guide
+  - Modeling applications can operate independently outside of the automation clusters.
+- Service meshes are presently unsupported, and we advise against their use for the setup.
+
+## Considerations
+
+Multi-region setups in itself bring their own complexity. The following items are such complexities and are not considered in our guides.
+You should familiarize yourself with those before deciding to go for a dual-region setup.
+
+- Managing multiple Kubernetes clusters and their deployments across regions
+- Monitoring and alerting
+- Increased costs of multiple clusters and cross-region traffic
+- Data consistency and synchronization challenges (for example brought in by the increased latency)
+  - bursts of increased latency can already have an impact
+- Managing DNS and incoming traffic
+
+## Region Loss
+
+In a dual-region setup, a loss of a region will invariably affect Camunda 8, regardless of whether it's the active or passive region.
+
+It means that the Zeebe stretch cluster will not have a quorum when half its brokers are not reachable anymore and will stop processing any new data. This will also affect the WebApps since they can not update or push new workflows. Essentially, this means that the workflow engine to halt until the region failover procedure is completed.
+
+The [operational procedure](<!-- TODO: link -->) looks in detail at how to temporarily recover from a region loss and ultimately how to fully reestablish the lost region. The procedure works the same way for active or passive region loss since we don't consider traffic routing (DNS) in the scenario.
```

Comment:
```suggestion
The [operational procedure](<!-- TODO: link -->) looks in detail at how to short-term recover from a region loss and how to long-term fully reestablish the lost region. The procedure works the same way for active or passive region loss since we don't consider traffic routing (DNS) in the scenario.
```